product_docs/docs/edbcloud/beta/using_cluster/06_analyze_with_superset.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Analyzing Your Data with Apache Superset"
+---
+You can use Apache Superset to analyze, explore, and visualize data stored in your PostgreSQL clusters.
+You can open Superset from the [EDB Cloud](https://portal.edbcloud.com/) portal using the **Analyze** links. 
+
+Default permissions include the ability to view data and create dashboards. To add or alter data sources, you need additional permissions. For more information on permissions to access Superset, see [Managing Superset Access](../administering_cluster/01a_superset_access).
+
+!!! Note
+    Apache Superset shares Single Sign On with EDB Cloud. If you log out of Superset you will be logged out of EDB Cloud and vice-versa.
+
+
+## Connecting Superset to your Cluster
+
+
+
+
+In addition to your password, you need your user, host, port, and database name to connect to Superset. To find this additional information for your cluster:
+
+1. Sign in to the [EDB Cloud](https://portal.edbcloud.com/) portal.
+
+3. Go to the [Clusters](https://portal.edbcloud.com/clusters) page.
+4. Select the name of your cluster.
+5. Select the **Connect** tab.
+
+!!!note
+Superset encyrpts your password.
```

Comment:
I think we need to be a bit clearer here.

```suggestion
Your password is always encrypted before storage and never leaves your cloud environment - it's used only by the Superset software running in your EDB Cloud infrastructure. As a defense-in-depth mechanism, EDB Cloud recommends using a Postgres user dedicated to Superset with a minimal set of read-only privileges to just the database you're connecting. Never use your `edb_admin` user with Superset.
```

@vishalsawale9: Can users always use a read-only credential for Superset? Can they even write data with it at all?

Reply:
> Can users always use a read-only credential for Superset? Can they even write data with it at all?

I think there is no harm using `edb_admin` user in superset for admin roles. Because `superset:admin` user can control the DML queries access on particular databases. The "Allow DML" checkbox is viewable only to admin. 

<img width="754" alt="Screenshot 2021-09-24 at 11 43 02 AM" src="https://user-images.githubusercontent.com/70691901/134626901-72b3580a-1ec8-451f-80e6-b53de5f31c9f.png">

So if admin makes any DB as Read-only (that is Allow checkbox unchecked), then Alpha and subsequent users cannot use any DML and sql lab shows something below:

<img width="701" alt="Screenshot 2021-09-24 at 11 46 55 AM" src="https://user-images.githubusercontent.com/70691901/134627322-a38e78e1-7cfa-4ec1-92bd-c533a3e5f592.png">

Reply:
@ba-edb , let me know if you think we should amend the note.

Reply:
Awesome, thanks Vishal.

@drothery-edb: Let's strip the phrase "read-only" from the doc, but publish it otherwise. We still don't want customers to be using `edb_admin` for data access generally, so I'd like to keep that guidance in place.

Reply:
ok. I'll take  "read-only" out of this sentence in the note: As a defense-in-depth mechanism, EDB Cloud recommends using a Postgres user dedicated to Superset with a minimal set of read-only privileges to just the database you're connecting.